SPARCore/GADRemoteModel.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+
+@interface GADRemoteModel : NSObject
+
++ (void) fetchModelsWithParams:(NSURL * _Nonnull)baseURL
+               queryParameters:(NSDictionary * _Nonnull)queryParams
+              modelTransformer:(NSArray<GADRemoteModel *>*_Nonnull(^_Nonnull)
```

Comment:
We can `typedef` these completion block signatures to make it easier to read/write here. I'll make a ticket to do it later :D